docs/tutorials/stateful-application/cassandra.md

```diff
@@ -1,374 +1,134 @@
-
-- [_Pods_](/docs/user-guide/pods)
-- [ _Services_](/docs/user-guide/services)
-- [_Replication Controllers_](/docs/user-guide/replication-controller)
-- [_Stateful Sets_](/docs/concepts/workloads/controllers/statefulset/)
-- [_Daemon Sets_](/docs/admin/daemons)
-
-## Prerequisites
-
-This example assumes that you have a Kubernetes version >=1.2 cluster installed and running,
-and that you have installed the [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
-command line tool somewhere in your path.  Please see the
-[getting started guides](https://kubernetes.io/docs/getting-started-guides/)
-for installation instructions for your platform.
-
-This example also has a few code and configuration files needed.  To avoid
-typing these out, you can `git clone` the Kubernetes repository to your local
-computer.
-
-## Cassandra Docker
-
-The pods use the [```gcr.io/google-samples/cassandra:v12```](https://github.com/kubernetes/examples/blob/master/cassandra/image/Dockerfile)
-image from Google's [container registry](https://cloud.google.com/container-registry/docs/).
-The docker is based on `debian:jessie` and includes OpenJDK 8. This image
-includes a standard Cassandra installation from the Apache Debian repo.  Through the use of environment variables you are able to change values that are inserted into the `cassandra.yaml`.
-
-| ENV VAR       | DEFAULT VALUE  |
-| ------------- |:-------------: |
-| CASSANDRA_CLUSTER_NAME | 'Test Cluster'  |
-| CASSANDRA_NUM_TOKENS  | 32               |
-| CASSANDRA_RPC_ADDRESS | 0.0.0.0          |
-
-## Quickstart
-
-If you want to jump straight to the commands we will run,
-here are the steps:
-
-```sh
-#
-# StatefulSet
-#
+{% capture overview %}
+This tutorial will show you how to develop a native cloud [Cassandra](http://cassandra.apache.org/) deployment on Kubernetes. In this instance, a custom Cassandra `SeedProvider` enables Cassandra to discover new Cassandra nodes as they join the cluster.
+{% endcapture %}
 
-# clone the example repository
-git clone https://github.com/kubernetes/examples
-cd examples
+{% capture objectives %}
+* Create and Validate a Cassandra headless service.
+* Use a StatefulSet to create a Cassandra ring.
+* Validate the StatefulSet.
+* Modify the StatefulSet.
+* Delete the StatefulSet and its pods.
+{% endcapture %}
 
-# create a service to track all cassandra statefulset nodes
-kubectl create -f cassandra/cassandra-service.yaml
+{% capture prerequisites %}
+You should already have a basic familiarity with [Pods](#), [Services](#), and [StatefulSets](#), but to complete this tutorial you only need: 
 
-# create a statefulset
-kubectl create -f cassandra/cassandra-statefulset.yaml
+* To [Install and Configure](/docs/tasks/tools/install-kubectl/) the `kubectl` command line
 
-# validate the Cassandra cluster. Substitute the name of one of your pods.
-kubectl exec -ti cassandra-0 -- nodetool status
+* To have a v1.5 or later Kubernetes Cluster running
```

Comment:
Avoid stating a version number, which will make keeping this doc up to date more difficult. I suggest just stating "Have a supported Kubernetes Cluster running".